protocol/protobuf/pairing.proto

```diff
@@ -255,3 +267,16 @@ message SyncContactRequestDecision {
     DECLINED = 1;
   }
 }
+
+/* `SyncProfile` is used to describe profile of logged in user */
+message SyncProfile {
+  string key_uid = 1;
+  string display_name = 2;
+  repeated SyncProfilePicture pictures = 3;
+}
+
+/* `SyncProfileSettings` is used to sync user preferences which are stored locally in the `settings` table 
+ * but for example we do not need to sync `display_name` which is one of the fields of `settings` table cause 
+ * we will already have it via `SyncProfile` */
+message SyncProfileSettings {
```

Comment:
Do we need an empty struct, we have already SyncSettings?

Reply:
That empty struct is more like predisposition for further changes, that's why it's empty. I wanted to do the all parts which need to be sync in the same way.

Reply:
Though I don't get this, why not use `SyncSetting` which we have already and add a new struct? 
I get that you'd like to have SyncProfile to collate similar infos (though I think we should re-use `SyncSetting` since it's there if we can, I believe display name is fine, but not sure about images @Samyoul ?)

If we want to do things the same way, we should reuse what we have for everything or extend it, instead of creating a new one, but maybe I misunderstand the purpose